packages/rgbpp/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export { remove0x } from '@rgbpp-sdk/ckb';
+export { ErrorCodes, ErrorMessages } from '@rgbpp-sdk/btc';
```

Comment:
Maybe errors in the BTC lib should also be renamed 
```suggestion
export { ErrorCodes as BtcErrorCodes, ErrorMessages as BtcErrorMessages } from '@rgbpp-sdk/btc';
```

Reply:
Okay, maybe if each sub-package is exported from its own path, the issue can also be resolved.